source/triggers/scheduled-triggers.txt

```diff
@@ -1,4 +1,5 @@
 .. _scheduled-trigger:
+.. _scheduled-triggers:
```

Comment:
I got an odd autobuilder error for a missing ref to `scheduled-triggers` (plural). The link works on the deployed page and neither this nor the page containing the ref have been touched in over a year, but _just in case_ I added this...